### PR TITLE
Update CreateTimelineFromJSONorCSV.md

### PR DESCRIPTION
**Pull request by deralexxx**
As mentioned in https://github.com/google/timesketch/issues/701 for the time the bug exists it should be mentioned in the readme

### CODE REVIEW

File: docs/CreateTimelineFromJSONorCSV.md [hunk 1]
```diff
@@ -42,8 +42,28 @@ Unlike JSON files, imports in JSONL format can be streamed from disk, making the
...
+    
+## Upload via HTTPS
+
+You can also upload the file directly via Web UI if enabled in /etc/timesketch.conf:
+
+``
+UPLOAD_ENABLED = True
+``
+
+### Known issue
```
Comment (berggren) [inline docs/CreateTimelineFromJSONorCSV.md]: Can we just link to the documentation for this?
https://github.com/google/timesketch/blob/master/docs/EnablePlasoUpload.md

Comment (deralexxx) [reply docs/CreateTimelineFromJSONorCSV.md]: hm indeed

Actually the EnablePlasoUpload could be refactored to EnableUpload.

Comment (berggren) [reply docs/CreateTimelineFromJSONorCSV.md]: Ok, I'll change the name to EnableUpload. Closing this one out.

**Comment by berggren** [comment]
Closing out as agreed. This is already present in the current documentation.

**Pull request closed without merging**